tripzone/src/components/Header/Header.js

```diff
@@ -0,0 +1,102 @@
+import "./Header.scss";
+import { useState, useEffect } from "react";
+
+function Header() {
+  const [sessionTime, setSessionTime] = useState(0);
+  const [loggedInTime, setLoggedInTime] = useState(null);
+  const [user, setUser] = useState([]);
+
+  useEffect(() => {
+    const storedUser = localStorage.getItem("user");
```

Comment:
could you maintain user info using react concept instead of local storage 
or local storage is used considering page load ?

Reply:
Local Storage is used considering page reload as context is being re-initialized and the user data is lost